CoffeeMachineSimulator/CoffeeMachineSimulator.Tests/Services/CoffeeServiceTests.cs

```diff
@@ -33,11 +37,52 @@ public void GetCoffees_Returns_ListOfCoffeeModels()
         public void AddNullCoffee_Throws_NewException()
         {
             var ex = Assert.Throws<Exception>(() => coffeeService.AddCoffee(null));
-            
+
             Assert.AreEqual(ex.Message, "You should not add null entries!");
         }
 
         //TODO: Add Unit test for Remaining add cases
+        [Test]
+        public void AddCoffeWithoutPrice_WillNotAddCoffe()
+        {
+            var initial_Length_List = coffeeService.GetCoffees().Count;
+            var AddDetailsCoffe = new CoffeeModel();
```

Comment:
please use camelCase instead of PascalCase. In c# this is a naming convention for creating new variables (not applicable for constants) : https://www.mooclab.club/threads/what-is-the-difference-between-pascal-case-and-camel-case.5636/